### DISCUSSION TITLE
Specific server

### DISCUSSION THREAD

**Discussion by dozer1dave** [tmknight/docker-nordvpn#25] [Q&A] [answered]
How would I request a specific server?  There used to be a CONNECT env variable that I could set to us4987 or whatever specific server I wanted.  Thanks

**Comment by tmknight** [comment]
https://github.com/tmknight/docker-nordvpn#:~:text=used%20by%20CHECK_CONNECTION_INTERVAL-,CONNECTION_FILTERS,-Use%20the%20NordVPN

**Reply by dozer1dave** [reply]
I saw that page but it is really vague what valid filters are.  The example of "filters[country_city_id]=8980922" will get me a specific server in Charlotte but how would I query to find the country_city_id for specific server us4987?  I also would be interested in other valid filters.  I tried [ip_address] and [id] with no luck.

**Comment by tmknight** [comment]
The link to a discussion of the API, and crafting filters, is also [depicted there](https://sleeplessbeastie.eu/2019/02/18/how-to-use-public-nordvpn-api/).

That said, the filters are used against the recommended server list and there isn't a filter (that I have found) for a specific server.  The scripting logic finds the lowest load server leveraging filters such as country, city (NordLynx) or VPN technology (p2p, CyberSec, etc.) for OpenVPN.

Help me understand the use case for wanting to always connect to a specific server, that could in fact be offline.

**Reply by dozer1dave** [reply]
This is resolved with the CONNECT variable.  But to answer your question, any Nord customers with dedicated IPs must connect to a specific server in order to receive their dedicated IP.

**Reply by tmknight** [reply]
Ah, yes...forgot about that - thanks

**Comment by tmknight** [comment, marked as answer]
Reinstates CONNECT: [Release 2.3.0-rc.1](https://github.com/tmknight/docker-nordvpn/releases/tag/2.3.0-rc.1)
